src/airflow_tools/filesystems/impl/sftp_filesystem.py

```diff
@@ -31,3 +31,6 @@ def delete_prefix(self, prefix: str):
 
     def list_files(self, prefix: str) -> list[str]:
         return self.hook.list_directory(prefix)
+    
+    def check_prefix(self, prefix: str) -> bool:
+        return bool(self.hook.list_directory(prefix))
```

Comment:
I think we can also use `path_exists` https://airflow.apache.org/docs/apache-airflow-providers-sftp/stable/_api/airflow/providers/sftp/hooks/sftp/index.html#airflow.providers.sftp.hooks.sftp.SFTPHook.path_exists